js/templates/ratingStars.html

```diff
@@ -0,0 +1,15 @@
+<% for (var i=0; i < ob.rating; i++) { %>
+<div class="inlineBlock positionRelative">
+  <% if (Math.floor(ob.rating) === i) { %>
+    <span class="ion-android-star textOpacity35"></span>
+    <span class="ion-android-star textOpacity1 positionAbsolute noOverflow" style="top:0;bottom:0;left:0;right:0;width:<%= ((ob.rating % 1) * 100) + '%'%>"></span>
+  <% } else { %>
+    <span class="ion-android-star textOpacity1"></span>
+  <% } %>
+</div>
+<% } %>
+<% for (var i=0; i < 5 - ob.rating; i++) { %>
+<div class="inlineBlock positionRelative">
+  <span class="ion-android-star textOpacity35"></span>
+</div>
+<% } %>
```

Comment:
If you look at the transactionModal model, there's a star class that might be useful here. You don't need to change it, I'm just noting it in case we want to update it later.